WhenWorkingWithASetOfPets.java

```diff
@@ -0,0 +1,50 @@
+package serenitylabs.tutorials.vetclinic.collections.exercises;
+
+import com.google.common.collect.Sets;
+import com.google.common.primitives.Booleans;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasItems;
+
+public class WhenWorkingWithASetOfPets {
+
+    @Test
+    public void should_add_Fido_to_the_set_of_pets() {
+        Set<String> names = Sets.newHashSet();
+
+        names.add("Fido");
+        assertThat(Boolean.TRUE, equalTo(names.contains("Fido")));
```

Comment:
Try to avoid comparing anything to a boolean, because the error message won't tell you what you were really trying to do.  In this case, you should use the Hamcrest collections matchers, e.g.:

```
        assertThat(names, contains("Fido"));
```